makechrootpkg.in

```diff
@@ -399,5 +407,15 @@ if (( ret != 0 )); then
 		die "Build failed, check %s/build" "$copydir"
 	fi
 else
+	if (( run_checkpkg )); then
+		msg "Running checkpkg"
+		msg2 "Downloading current versions"
+		if pacman --noconfirm -Swdd --logfile /dev/null "${pkgnames[@]}"; then
```

Comment:
If you want to be fancy, you could check with `pacman -Sddp` whether it is already downloaded, and skip the `-Swdd` integrity checks plus the log message.

Otherwise this seems fine.

Reply:
will do in a followup patch, as i would like to append all pendings things and, as $pkgname is an array, `-Swdd` would be a small fiddling with results returned